examples/examples.py

```diff
@@ -23,8 +23,8 @@
         "study_case": "dam_with_complex_clearing",
     },
     "small_with_BB_and_LB": {
-        "scenario": "example_01d",
-        "study_case": "dam_with_complex_clearing",
+        "scenario": "example_01c",
```

Comment:
did you merge c and d? and also I saw you added a new e example, is it also in the dict of available examples now?